### PR TITLE
(976, 964) Bugfixes

### PR DESCRIPTION
**Pull request by pezholio**
A couple of bugfixes for the following cards:

- https://trello.com/c/QRMBqJwQ/964-error-when-selecting-no-additional-needs
- https://trello.com/c/jpKlVXAv/976-check-your-answers-screen-missing-task-names

### CODE REVIEW

**Comment by pezholio** [comment]
[Check Your Answers screen missing task names](https://trello.com/c/jpKlVXAv/976-check-your-answers-screen-missing-task-names)

**Comment by pezholio** [comment]
[Error when selecting no additional needs](https://trello.com/c/QRMBqJwQ/964-error-when-selecting-no-additional-needs)

File: server/views/applications/pages/check-your-answers/review.njk [hunk 1]
```diff
@@ -114,11 +114,11 @@
       {% for section in checkYourAnswersSections(page.application) %}
         <h2 class="govuk-heading-l">{{ section.title }}</h2>
         {% for task in section.tasks %}
-          <div class="main-box">
+          <div class="main-box" data-cy-check-your-answers-section="{{ task.id }}">
             <div class="box-title govuk-heading-m">
```
Comment (richpjames) [inline server/views/applications/pages/check-your-answers/review.njk:118]: Not sure but I wonder if a heading tag might be more relevant for the titles?

**Review by richpjames — APPROVED** [review]

File: server/views/applications/pages/check-your-answers/review.njk (hunk 1 above)
Comment (pezholio) [reply server/views/applications/pages/check-your-answers/review.njk:118]: That's probably true, I was going off the original designs

Comment (pezholio) [reply server/views/applications/pages/check-your-answers/review.njk]: Done this now!

**Pull request merged**